src/main/java/com/google/devtools/build/lib/packages/Globber.java

```diff
@@ -39,11 +39,12 @@ Token runAsync(
       throws BadGlobException, InterruptedException;
 
   /**
-   * Fetches the result of a previously started glob computation. The returned list must be ordered
-   * deterministically. For more obvious correctness, implementations should generally sort the list
-   * they return.
+   * Fetches the result of a previously started glob computation. The returned list has an arbitrary
+   * order. For more obvious correctness, implementations should generally sort the list they
```

Comment:
Let's just delete the second sentence ("For more obvious correctness..."). Or if you want you can rephrase to something like "Consumers probably want to sort this list".

Reply:
deleted